packages/framework/bundle/publisher2/bundle.js

```diff
@@ -95,7 +95,7 @@ Oskari.clazz.define("Oskari.mapframework.bundle.publisher2.PublisherBundle", fun
         },
         {
             "type": "text/javascript",
-            "src": "../../../../bundles/framework/publisher2/tools/ScalebarTool.js"
+            "src": "../../../../bundles/mapping/mapmodule/plugin/scalebar/publisher/ScalebarTool.js"
```

Comment:
This is kinda weird with published picking some parts under mapmodule. It helps with code not being packaged for the client when publisher isn't present though (embedded maps for example). 

How about if we create a `mapmodule/publisher/tools.js` that only imports the tools that mapmodule is providing for the publisher:
```
import '../plugin/scalebar/publisher/ScalebarTool.js';
import '../plugin/logo/publisher/LogoTool.js';
```
Publisher would then have some kind of a documentable "API" how it could in some cases pick tools from other bundles?

Reply:
The src here would then be just one line 
```suggestion
            "src": "../../../../bundles/mapping/mapmodule/publisher/tools.js"
```

Reply:
Sounds reasonable. Let's move to this direction. 👍